shared/django_apps/codecov_auth/models.py

```diff
@@ -751,32 +780,6 @@ class Meta:
     )
 
 
-class Account(BaseModel):
-    name = models.CharField(max_length=100, null=False, blank=False, unique=True)
-    is_active = models.BooleanField(default=True, null=False, blank=True)
-    plan = models.CharField(
-        max_length=50,
-        choices=PlanName.choices(),
-        null=False,
-        default=PlanName.BASIC_PLAN_NAME.value,
-    )
-    plan_seat_count = models.SmallIntegerField(default=1, null=False, blank=True)
-    free_seat_count = models.SmallIntegerField(default=0, null=False, blank=True)
-    plan_auto_activate = models.BooleanField(default=True, null=False, blank=True)
-    is_delinquent = models.BooleanField(default=False, null=False, blank=True)
-    users = models.ManyToManyField(
-        User, through="AccountsUsers", related_name="accounts"
-    )
-
-    class Meta:
-        ordering = ["-updated_at"]
-        app_label = CODECOV_AUTH_APP_LABEL
-
-    def __str__(self):
-        str_representation_of_is_active = "Active" if self.is_active else "Inactive"
-        return f"{str_representation_of_is_active} Account: {self.name}"
-
-
```

Comment:
had to move this model's position in the file so that `Owner` could reference `Account`